### PR TITLE
Allowing ping HEAD requests to the target status resources

### PR DESCRIPTION
**Pull request by achow-zenoss**

### CODE REVIEW

File: src/conf/zenoss-zapp-ping.lua [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+local common = loadfile("conf/zproxy-common.lua")
+common()
+local uri = ngx.var.uri
+local uri_prefix = extract_prefix(uri)
+local frontend = extract_frontend(ngx.var.http_host)
+local domain_name = extract_domain(frontend)
+local backend = extract_backend(uri_prefix, frontend, domain_name)
+local req_headers = ngx.req.get_headers()
+local auth_token = extract_auth_token(req_headers)
```
Comment (jplouis) [inline src/conf/zenoss-zapp-ping.lua]: Don't need to extract auth tokens.

Comment (achow-zenoss) [inline src/conf/zenoss-zapp-ping.lua]: Yes. removed...

File: src/conf/zenoss-zapp-ping.lua [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+--]]----------------------------------------------------------------------------
+
+local common = loadfile("conf/zproxy-common.lua")
+common()
+local uri = ngx.var.uri
+local uri_prefix = extract_prefix(uri)
+local frontend = extract_frontend(ngx.var.http_host)
+local domain_name = extract_domain(frontend)
+local backend = extract_backend(uri_prefix, frontend, domain_name)
+local req_headers = ngx.req.get_headers()
```
Comment (jplouis) [inline src/conf/zenoss-zapp-ping.lua]: req_headers is not used

Comment (achow-zenoss) [inline src/conf/zenoss-zapp-ping.lua]: You are right. fixing.

**Pull request merged**